src/lib/pubkey/dilithium/dilithium.h

```diff
@@ -0,0 +1,128 @@
+/*
+* Crystals Dilithium Digital Signature Algorithms
+* Based on the public domain reference implementation by the
+* designers (https://github.com/pq-crystals/dilithium)
+*
+* Further changes
+* (C) 2021-2022 Jack Lloyd
+* (C) 2021-2022 Manuel Glaser and Michael Boric, Rohde & Schwarz Cybersecurity
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#ifndef BOTAN_DILITHIUM_COMMON_H_
+#define BOTAN_DILITHIUM_COMMON_H_
+
+#include <botan/der_enc.h>
+#include <botan/exceptn.h>
+#include <botan/pk_keys.h>
+
+#include <botan/internal/pk_ops_impl.h>
+
+#if !defined(BOTAN_HAS_DILITHIUM)
+static_assert(false, "botan module 'dilithium' is useful only when enabling modules 'dilithium'");
+#endif

```

Comment:
This is not needed. I guess you took some inspiration from Kyber:
https://github.com/randombit/botan/blob/2fb7ba81b0304f0864a557e3f3a010f4c2ce08d3/src/lib/pubkey/kyber/kyber_common/kyber.h#L23-L25

For Kyber we introduced that to make the library build fail gracefully if a user enables `kyber_common` but neither `kyber` nor `kyber_90s`. That would result in compiling the "abstract" kyber algorithm but no concrete implementation of the kyber primitives for modern or 90s kyber.

Please just remove this,